Sources/AuthFoundation/JWT/Internal/DefaultAccessTokenValidator.swift

```diff
@@ -0,0 +1,49 @@
+//
+// Copyright (c) 2022-Present, Okta, Inc. and/or its affiliates. All rights reserved.
+// The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+//
+// You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+// WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//
+// See the License for the specific language governing permissions and limitations under the License.
+//
+
+import Foundation
+
+#if canImport(CommonCrypto)
+import CommonCrypto
+#endif
+
+struct DefaultAccessTokenValidator: AccessTokenValidator {
+    func validate(accessToken: String, idToken: JWT) throws {
+        #if !canImport(CommonCrypto)
+        throw JWTError.signatureVerificationUnavailable
+        #endif
+
+        guard let atHash = idToken.value(String.self, for: "at_hash")
+        else {
+            return
```

Comment:
Can you add a test for this case? If the `at_hash` is missing from the ID Token, it should be valid. (This happens for other IdPs like Auth0.